### PR TITLE
Fix bug that does not reset all recipes page after search

### PR DESCRIPTION
**Pull request by zwsaile**
#### What's this PR do?
- Fixes bug that keeps search results on all recipes page after leaving search results page

#### Where should the reviewer start?
- scripts.js

#### How should this be manually tested?
- Run a search and leave the all recipes page, navigate back to all recipes, and determine if all recipes are displayed

### CODE REVIEW

File: src/classes/RecipeRepository.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export class RecipeRepository {
   constructor(recipes) {
     this.recipes = recipes;
+    this.filteredRecipes;
```
Comment (sted1994) [inline src/classes/RecipeRepository.js:4]: Lets remove this key

File: src/classes/RecipeRepository.js [hunk 2]
```diff
@@ -36,6 +37,11 @@ getRecipesBySearch(input) {
   })
   this.recipes = recipesByName
 }
+
+  getAllRecipes(recipes) {
```
Comment (sted1994) [inline src/classes/RecipeRepository.js:41]: Good fix

**Review by sted1994 — APPROVED** [review]
Nice job!

File: src/scripts.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ let currentRecipe;
 let ingredients;
 let usersData;
 let randomUser
+let recipeDataClasses
```
Comment (Wperricone) [inline src/scripts.js:27]: Great!

**Review by Wperricone — APPROVED** [review]

**Pull request merged**